plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -85,7 +85,8 @@ tests =
   , goldenWithEval "Shows a kind with :kind" "T12" "hs"
   , goldenWithEval "Reports an error for an incorrect type with :kind" "T13" "hs"
   , goldenWithEval' "Returns a fully-instantiated type for :type" "T14" "hs" (if ghcVersion >= GHC98 then "ghc98.expected" else "expected") -- See https://gitlab.haskell.org/ghc/ghc/-/issues/24069
-  , knownBrokenForGhcVersions [GHC92, GHC94, GHC96, GHC98] "type +v does not work anymore with 9.2" $ goldenWithEval "Returns an uninstantiated type for :type +v, admitting multiple whitespaces around arguments" "T15" "hs"
+  , knownBrokenForGhcVersions [GHC92, GHC94, GHC96, GHC98, GHC910] "type +v does not work anymore with 9.2" $
```

Comment:
It seems to me that we should just remove this testcase, since all supported GHCs now say:
```
:type +v 123
`:type +v' has gone; use `:type' instead
```

Reply:
Removed.